.github/workflows/ci.yaml

```diff
@@ -35,9 +35,12 @@ jobs:
     name: commit-lint
     runs-on: ubuntu-latest
     steps:
-      - uses: ahmadnassri/action-commit-lint@v1
+      - uses: actions/checkout@v3
         with:
-         config: conventional
+          fetch-depth: 0
```

Comment:
If you use fetch depth 0 how is the commitlint action ever going to check older commits?

Reply:
See [readme of the checkout action](https://github.com/actions/checkout): 0 means all commits